analysis/tests/lighttpd/src/lighttpd-minimal.rs

```diff
@@ -50,18 +50,55 @@ pub struct fdnode_st {
 
 pub type fdnode = fdnode_st;
 
-unsafe extern "C" fn connection_close(mut srv: *mut server, mut con: *mut connection) {}
+unsafe extern "C" fn fdnode_init() -> *mut libc::c_void /*TODO: handle *mut fdnode */ {
```

Comment:
For all of these changes, I want to land a `lighttpd` minimal test for `c2rust-analyze` in CI first, just checking that it doesn't crash on this as input.  We can merge this earlier if it's blocking anything and I don't get to adding the test in time, but hopefully I'll be able to.

Reply:
I believe #757 does this now

Reply:
I'll add that test now (it should be pretty simple).  #757 doesn't quite do it, because then it runs `FileCheck`, which we don't need to run.  `FileCheck` isn't set up to run in CI yet, so avoiding that if we don't need it is helpful.